packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.module.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+
+import routing from './add.routing';
+import component from './add.component';
+import service from '../models.service';
+import capabilities from '../../../capabilities.service';
+
+import storageService from '../../../../storages/containers/containers.service';
```

Comment:
Maybe you could just have a dependency to `ovhManagerPciStoragesContainers` module to reuse the `PciProjectStorageContainersService` service (cf : https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/containers/index.js)